src/ccl/charts/chart.template.html

```diff
@@ -0,0 +1,14 @@
+<div class="chart" *ngFor="let chart of chartList">
```

Comment:
This encapsulation feels weird. I would expect the `charts` directive to simply loop and display a list of `chart` components. That way the settings stuff can be encapsulated inside `chart` rather than having it live here. `chart` can still contain `line-graph`, I like that that lives separately from the chart's settings.